data/mods/Aftershock/spells/spells.json

```diff
@@ -54,5 +54,97 @@
     "base_casting_time": 1,
     "min_duration": 100,
     "max_duration": 100
+  },
+  {
+    "id": "cream_prot_cold",
+    "type": "SPELL",
+    "name": "heat retention cream",
+    "//": "Used for heat retention cream not castable",
+    "description": "You rub the thick cream all over your body in preparation for the expected cold.",
+    "valid_targets": [ "self" ],
+    "flags": [ "SILENT", "NO_LEGS", "NO_HANDS" ],
+    "effect": "spawn_item",
+    "effect_str": "cold_res_cream",
+    "shape": "blast",
+    "max_level": 0,
+    "min_damage": 0,
+    "max_damage": 0,
+    "damage_increment": 0,
+    "damage_type": "biological",
+    "difficulty": 0,
+    "spell_class": "NONE",
+    "base_casting_time": 6000,
+    "base_energy_cost": 2,
+    "min_duration": 720000,
+    "max_duration": 720000
+  },
+  {
+    "id": "cream_greater_prot_cold",
+    "type": "SPELL",
+    "name": "heat retention cream xtreme",
+    "//": "Used for heat retention cream not castable",
+    "description": "You rub the thick cream all over your body in preparation for unbearable cold.",
+    "valid_targets": [ "self" ],
+    "flags": [ "SILENT", "NO_LEGS", "NO_HANDS" ],
+    "effect": "spawn_item",
+    "effect_str": "cold_res_cream_greater",
+    "shape": "blast",
+    "max_level": 0,
+    "min_damage": 0,
+    "max_damage": 0,
+    "damage_increment": 0,
+    "damage_type": "biological",
+    "difficulty": 0,
+    "spell_class": "NONE",
+    "base_casting_time": 6000,
+    "base_energy_cost": 2,
+    "min_duration": 720000,
+    "max_duration": 720000
+  },
+  {
+    "id": "protection_from_cold",
+    "type": "SPELL",
+    "name": "emergency insulation",
+    "//": "Used for cbm not castable",
+    "description": "You feel a jolt through your body as energy pulses through the wires in your flesh.",
+    "valid_targets": [ "self" ],
+    "flags": [ "SILENT", "NO_LEGS", "NO_HANDS", "NO_PROJECTILE" ],
+    "effect": "attack",
+    "effect_str": "afs_cold_protect",
+    "shape": "blast",
+    "max_level": 0,
+    "spell_class": "NONE",
+    "base_casting_time": 1,
+    "min_duration": 1000,
+    "max_duration": 1000
+  },
+  {
+    "id": "cryo_blast",
+    "type": "SPELL",
+    "name": "cryokinetic blast",
```

Comment:
Change this to   "name": { "str": "cryokinetic blast" }, 

As the monster spell cannot detect cryo_blast if there isn't a string there

Reply:
Thank you for running this down.  I was getting so frustrated running into the same wall over and over.